dazzle-rewrite/chunks/lang-c/Dockerfile

```diff
@@ -2,11 +2,14 @@ ARG base
 FROM ${base}
 
 USER root
+
 # Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
-ENV TRIGGER_REBUILD=1
-RUN curl -o /var/lib/apt/dazzle-marks/llvm.gpg -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key \
-    && apt-key add /var/lib/apt/dazzle-marks/llvm.gpg \
-    && echo "deb https://apt.llvm.org/focal/ llvm-toolchain-focal main" >> /etc/apt/sources.list.d/llvm.list \
+ENV TRIGGER_REBUILD=3
```

Comment:
did we need this change? The build gets retriggered in the beow step due to change in the step